### PR TITLE
Normalize

### PR DESCRIPTION
**Pull request by johangiraud**

### CODE REVIEW

File: styleguide.css [hunk 1]
```diff
@@ -135,6 +139,15 @@ summary {
     color: #00283C;
 }
 
+.icon-logo-cafebabel .path1:before {
```
Comment (vinyll) [inline styleguide.css:142]: What are these for?

**Review by vinyll — APPROVED** [review]

File: styleguide.css (hunk 1 above)
Comment (johangiraud) [reply styleguide.css:142]: told you it was the way for icomoon to have 1 icon with severals colors

Comment (vinyll) [reply styleguide.css:142]: Didn't you end up making the logo in SVG so you have a proper logo and you don't need to tweak this?

Comment (johangiraud) [reply styleguide.css:142]: yep

Comment (vinyll) [reply styleguide.css:142]: Ok, that's why I was surprise to see these lines coming up again and was confused.
This PR being closed I guess we need to clean this in a new PR ;)

Comment (johangiraud) [reply styleguide.css:142]: ok

**Pull request merged**